tests/rkt_net_test.go

```diff
@@ -338,18 +338,23 @@ func TestNetDefaultRestrictedConnectivity(t *testing.T) {
 	f("--net=default-restricted")
 }
 
-func testPortFwdConnectivity(ctx *testutils.RktRunCtx, t *testing.T, httpGetIP string, rktArg string, shouldSucceed bool, bannedPorts map[int]struct{}) {
+func testPortFwdConnectivity(ctx *testutils.RktRunCtx, t *testing.T, httpGetIP string, httpServePort int, rktArg string, shouldSucceed bool, bannedPorts map[int]struct{}) {
 	httpPort, err := testutils.GetNextFreePort4Banned(bannedPorts)
 	if err != nil {
 		t.Fatalf("%v", err)
 	}
 	bannedPorts[httpPort] = struct{}{}
 
-	httpServeAddr := fmt.Sprintf("0.0.0.0:%d", httpPort)
+	if httpServePort == 0 {
+		httpServePort = httpPort
+	}
+
+	httpServeAddr := fmt.Sprintf("0.0.0.0:%d", httpServePort)
 	testImageArgs := []string{
-		fmt.Sprintf("--ports=http,protocol=tcp,port=%d", httpPort),
+		fmt.Sprintf("--ports=http,protocol=tcp,port=%d", httpServePort),
 		fmt.Sprintf("--exec=/inspect --serve-http=%v", httpServeAddr),
 	}
+	t.Logf("testImageArgs: %v", testImageArgs)
```

Comment:
testImageArgs could be named testImagePatches since it is not really the parameters given to the executable in the image, but the changes on the image performed by `actool patch-manifest`.

The log could be clearer:
```
t.Logf("Listening on %v", httpServeAddr)
```